package.json

```diff
@@ -1,11 +1,11 @@
-
 {
   "name": "jokes-app",
```

Comment:
The name is probably from the copied files ;)

Reply:
👍 thx